charts/alfresco-sync-service/values.yaml

```diff
@@ -17,7 +17,7 @@ podSecurityContext:
   fsGroup: 1000
 image:
   repository: quay.io/alfresco/service-sync
-  tag: 5.0.0
+  tag: 4.0.1
```

Comment:
should stay on 5.x since it's latest GA

Reply:
good point, the 4.x enforcement for acs 23.2 needs to happen in the umbrella chart 👍

Reply:
I remember reverting this manually last time we did the release, there is not an easy way to handle it via updatecli matrix